content/git/concepts/branch/branch.md

```diff
@@ -36,6 +36,18 @@ git branch -d <branch-name>
 
 As long as the branch has no uncommitted code, this command will remove the branch entirely. It will no longer be available to check out, and the changed code will be unrecoverable.
 
+Sometimes Git will refuse to delete a branch because it has changes that have not been merged into the upstream or remote version of that branch. In these cases, Git can be _forced_ to delete the branch by changing the flag to a capital "D" or adding the `--force` flag like so:
+
+```shell
+# Option 1:
+git branch -D <branch-name>
+
+# Option 2:
+git branch -d --force <branch-name>
```

Comment:
```suggestion
# Option 1:
$ git branch -D my-branch

# Option 2:
$ git branch -d --force my-branch
```

After the syntax of a command is introduced, instead of using `<placeholder-syntax>` let's switch to using fictional names like `my-branch`.

Reply:
I don't know about having the `$` in front of the comments. Those commented lines are not commands that should be entered into the command line. If anything, maybe it is better that they get moved out of the code snippet altogether if each line has to be preceded by `$`